code/modules/cm_marines/overwatch.dm

```diff
@@ -1049,12 +1049,24 @@
 		if("move")
 			message = pick(";GET MOVING!", ";GO, GO, GO!", ";WE ARE ON THE MOVE!", ";MOVE IT!", ";DOUBLE TIME!")
 			say(message)
+			var/image/move = image('icons/mob/talk.dmi', icon_state = "order_move")
+			overlays += move
+			spawn(5 SECONDS)
```

Comment:
I know this is used everywhere here, but `spawn()` is actually a pretty expensive proc since it copies every variable and state of the parent proc, duplicating it.
The way /tg/ solves is it by a very elegant timer subsystem that requires other things to port, but for now we can use `set waitfor = FALSE` and `sleep()` instead of `spawn()` instead.